src/DotvvmAcademy.Web/Course/010_concepts/en/60_conclusion.md

```diff
@@ -9,15 +9,15 @@ Solution: .solution
 
 # Conclusion
 
-Congratulations! You learned the basic concepts of DotVVM!
+Congratulations! You have learned the basic concepts of DotVVM!
 
 ---
 
 ## Summary
 
-- DotVVM is an MVVM web framework.
-- Views are what the users interact with.
-- ViewModels contain data and logic for Views to use.
-- Dothtml is made (roughly) of Directives, Bindings, and Controls.
+- Every page in DotVVM consists of the View and the ViewModel. We are using the MVVM design pattern.
```

Comment:
Since Views/ViewModels denote a class of files, "a View" and "a ViewModel" is appropriate.